db/migrate/20230227143427_create_versions.rb

```diff
@@ -0,0 +1,31 @@
+# This migration creates the `versions` table, the only schema PT requires.
```

Comment:
Shame it's called 'versions' but I guess there's probably not much we can do about it!

Reply:
@DavidBiddle we can change it, any suggestions? I just wanted to keep the default settings until we start using it more and then tweak it further.

Reply:
Maybe something like `audits`, `audit-versions` or `history` - not sure really, just worried it might be mistaken for something to do with our form versioning. But I don't feel that stongly about it if we can't think of something good!

Reply:
I think if we use it for all papertail object versioning - as we currently do - we should use `versions` as it's the papertail default and is what people expect across projects.

I did look at [using a custom table/class for form versions](https://github.com/paper-trail-gem/paper_trail#6a-custom-version-classes) and calling it something like `editions`/`revisions`/`audit` as it seems such a fundamental part of the domain. In this case i think using a better name makes sense. 

Doing that makes using papertrail for other models more complicated and it's less flexible, so i'm not sure it's worth it